subsys/net/lib/wifi_mgmt_ext/wifi_mgmt_ext.c

```diff
@@ -100,7 +100,8 @@ static int __stored_creds_to_params(struct wifi_credentials_personal *creds,
 	/* Defaults */
 	params->security = creds->header.type;
 #ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
-	if (params->security == WIFI_SECURITY_TYPE_EAP_TLS) {
+	if (params->security == WIFI_SECURITY_TYPE_EAP_TLS ||
```

Comment:
better to make a helper `is_enterprise_security` as we are planning to extend this soon.